sandbox/apparmor/apparmor.go

```diff
@@ -481,6 +509,7 @@ var (
 
 	// Filesystem root defined locally to avoid dependency on the
 	// 'dirs' package
+	// XXX: is this still useful/relevant? The 'dirs' package is used here already.
```

Comment:
until the code stops using it is relevant

Reply:
I meant whether "avoid dependency on the 'dirs' package" is still relevant. Could we move to just use `dirs.GlobalRootDir`? I worked around it already so it's not that important for the time being, but it means tests have to be careful not to dig around the host filesystem.

Reply:
TODO: fix this eventually in follow-up PR